### ISSUE TITLE
No named parameter with the name 'autovalidate'. for Flutter 2.8 stable update 

### ISSUE THREAD

**Issue by theRoushan** [VictorRancesCode/json_to_form#16]
flutter/.pub-cache/hosted/pub.dartlang.org/json_to_form-0.0.4/lib/json_schema.dart:338:7: Error: No named parameter with the name 'autovalidate'.
      autovalidate: formGeneral['autoValidated'] ?? false,
      ^^^^^^^^^^^^
../../Downloads/flutter/packages/flutter/lib/src/widgets/form.dart:40:9: Context: Found this candidate, but the arguments don't match.
  const Form({
        ^^^^

`[](url)`
<img width="1155" alt="Screenshot 2021-12-14 at 1 15 11 PM" src="https://user-images.githubusercontent.com/52878060/145954672-ec5317f4-6fb3-4b22-8af8-b81d9773d80f.png">

**Comment by theRoushan** [comment]
This error comes in flutter 2.8 stable version

**Comment by gautamenbake** [comment]
yes @VictorRancesCode this is coming in latest flutter 2.8.1 version
Thanks for this wonderful package pls provide solution for this issue as soon as possible.

**Comment by arpanEnbake** [comment]
Any updates on the issue @VictorRancesCode ?
